packages/kununu-theme-v2/scss/base/type.scss

```diff
@@ -114,7 +104,7 @@ p {
 // Still used in profiles
 .text-right   { text-align: right; }
 
-// Still used in homepage, mykununu sidebar
+// Still used in mykununu sidebar
 .text-center  { text-align: center; }
```

Comment:
Where? I can't find it...

Reply:
Only in this file: https://github.com/kununu/app-mykununu/blob/master/src/components/App/__snapshots__/App.spec.jsx.snap

It's just a snapshot, but I'm not sure it's safe to be removed tho.

Reply:
Still can't find it... only `text-center-xs text-center-sm`